metoncofit/DEA/CCLE_DEA.R

```diff
@@ -0,0 +1,71 @@
+#### imports ####
```

Comment:
Don't worry about the comments I made right now. You can revisit these when we go back to the CCLE/TCGA MetOncoFit models.

Reply:
The more important comment is regarding log2 FC and z-score calc. Be sure to only choose 1 when we label DE genes.